runtime/tutor.txt

```diff
@@ -227,6 +218,138 @@ _________________________________________________________________
+=                      THE CHANGE COMMAND                       =
+=================================================================
+
+ Press c to change the current selection.
+
+ The change command deletes the current selection and enters
+ Insert mode, so it is a very common shorthand for di.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Move to the start of an incorrect word and press w to
+    select it.
+ 3. Press c to delete the word and enter Insert mode.
+ 4. Type the correct word.
+ 5. Repeat until the line matches the line below it.
+
+ --> This paper has heavy words behind it.
+     This sentence has incorrect words in it.
+
+
+
+
+=================================================================
+=                      COUNTS WITH MOTIONS                      =
+=================================================================
+
+ Type a number before a motion to repeat it that many times.
+
+ 1. Move the cursor to the line below marked -->.
+
+ 2. Type 2w to move 2 words forward.
+
+ 3. Type 3e to move to the end of the third word forward.
+
+ 4. Type 2b to move 2 words backwards
+
+ 5. Try the above with different numbers.
+
+ --> This is just a line with words you can move around in.
+
+
+
+
+
+=================================================================
+=                        SELECTING LINES                        =
+=================================================================
+
+ Press x to select a whole line. Press again to select the next.
+
+ 1. Move the cursor to the second line below marked -->.
+ 2. Press x to select the line, and d to delete it.
+ 3. Move to the fourth line.
+ 4. Press x twice or type 2x to select 2 lines, and d to delete.
+
+ --> 1) Roses are red,
+ --> 2) Mud is fun,
+ --> 3) Violets are blue,
+ --> 4) I have a car,
+ --> 5) Clocks tell time,
+ --> 6) Sugar is sweet,
+ --> 7) And so are you.
+
+
+
+
+=================================================================
+=                            UNDOING                            =
+=================================================================
+
+ Type u to undo. Type U to redo.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Move to the first error, and press d to delete it.
+ 3. Type u to undo your deletion.
+ 4. Fix all the errors on the line.
+ 5. Type u several times to undo your fixes.
+ 6. Type U (<SHIFT> + u) several times to redo your fixes.
+
+ --> Fiix the errors on thhis line and reeplace them witth undo.
+
+
+
+
+
+
+
+
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ * Type w to select forward until the next word.
+   * Type e to select to the end of the current word.
+   * Type b to select backward to the start of the current word.
+   * Use uppercase counterparts, W,E,B, to traverse WORDS.
```

Comment:
Should we stick these together?

Reply:
You mean like this?
```
* Type w to select forward until the next word.
  Type e to select to the end of the current word.
  Type b to select backward to the start of the current word.
  Use uppercase counterparts, W,E,B, to traverse WORDS.
```

Reply:
No, I just wonder why you stick `W,E,B,` together.

Reply:
Or do you mean put them all in one sentence?

Reply:
Oh, I did that just for the sake of having the keys together. Do you think I should change it?

Reply:
I think it might be confusing and the grammar is invalid, there should be spaces between comma.

Reply:
I did the same thing [here](https://github.com/helix-editor/helix/blob/58af5f2d513973eaf387713ba11386a81e67cf2a/runtime/tutor.txt#L159).

Reply:
I think it feels weird but it's fine I guess.